utils/keycodes.js

```diff
@@ -1,3 +1,10 @@
+/**
+ * Browser dependencies
+ */
+const { userAgent } = window.navigator;
+
+const isMac = userAgent.indexOf( 'Mac' ) !== -1;
```

Comment:
We use a different test elsewhere, and should seek to consolidate or create a shared utility:

https://github.com/WordPress/gutenberg/blob/75d81e9d13c065fc80d3184fe9ac58c0da0f0573/editor/edit-post/keyboard-shortcuts.js#L1

Reply:
Thanks, did not realise.

Reply:
Commented on https://github.com/WordPress/gutenberg/pull/3755#issuecomment-358300913. I wouldn't like us to have a lot of different keyboard combinations that are not rememberable.